src/odemis/driver/xt_client.py

```diff
@@ -790,6 +790,77 @@ def set_use_case(self, state):
             self.server._pyroClaimOwnership()
             return self.server.set_use_case(state)
 
+    def get_mpp_orientation(self):
+        """
+        Get the current multi probe orientation in degrees.
+
+        :return (float): Multi orientation in degrees
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.get_mpp_orientation()
+
+    def get_aperture_index(self):
+        """
+        Get the current aperture index.
+
+        :return (float): Aperture index (range 0 - 14)
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.get_aperture_index()
+
+    def set_aperture_index(self, aperture_idx):
+        """
+        Set the current aperture index.
+
+        :param aperture_idx:  (float): Aperture index (range 0 - 14)
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            self.server.set_aperture_index(aperture_idx)
+
+    def aperture_index_info(self):
+        """
+        Get the current aperture index information, contains the range (0 - 14).
+
+        :return (dict with a list of lenght 2): The range of the aperture index for the keyword "range".
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.aperture_index_info()
+
+    def get_beamlet_index(self):
+        """
+        Get the current beamlet index.
```

Comment:
Here you could maybe add some info that the beamlet index is characterized by two values in a grid.